gradle/Dockerfile

```diff
@@ -1,7 +1,43 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 
+# Install Java
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
I'm not a docker expert so let me know if there's any better ways to compose these changes.